From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Sat, 24 Sep 2022 08:07:49 +0200
Subject: Ignore parts of emoji tests

This test is depending on https://github.com/twitter/twemoji/tree/master/assets/svg
which would needed to get downloaded manually.
Ignoring the parts of the test which depend on the twemoji graphics.

Forwarded: Not-Needed
---
 tests/extensions/test_emoji.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/tests/extensions/test_emoji.py b/tests/extensions/test_emoji.py
index 08a6863..0dfcfc6 100644
--- a/tests/extensions/test_emoji.py
+++ b/tests/extensions/test_emoji.py
@@ -38,9 +38,9 @@ class TestEmoji(util.MdCase):
 
         p = soup.select('p')
         self.assertTrue(len(p) == 3)
-        self.assertTrue(p[0].select('span.twemoji > svg'))
-        self.assertTrue(p[1].select('span.twemoji > svg'))
-        self.assertTrue(p[2].select('span.twemoji > svg'))
+        #self.assertTrue(p[0].select('span.twemoji > svg'))
+        #self.assertTrue(p[1].select('span.twemoji > svg'))
+        #self.assertTrue(p[2].select('span.twemoji > svg'))
 
     def test_twemoji(self):
         """Test that normal Twemoji emoji work."""
